app/experimenter/experiments/admin/nimbus.py

```diff
@@ -24,11 +25,14 @@ def clean_feature_value(self):
         feature_value = self.cleaned_data["feature_value"]
         feature_config = self.cleaned_data["experiment"].feature_config
 
-        if feature_value and feature_config:
-            feature_schema = json.loads(
-                self.cleaned_data["experiment"].feature_config.schema
-            )
+        feature_schema = None
+        if feature_config and feature_config.schema:
+            try:
+                feature_schema = json.loads(feature_config.schema)
+            except json.JSONDecodeError:
+                pass
 
+        if feature_value and feature_schema:
```

Comment:
What's this? Is this related?

Reply:
Related insofar as I couldn't exercise the admin panel without it

Reply:
Oh yep yes, we should rip that whole thing out that's not needed anymore, I'll file a followup for that.